packages/Algernon.package/ALGCore.class/class/initializeGlobalKeyBindings.st

```diff
@@ -4,7 +4,7 @@ initializeGlobalKeyBindings
 	KeyBindings := Dictionary newFrom:
 		{ '<down>' -> [:anEvent :morph | morph selectNext]. 
 		'<up>' -> [:anEvent :morph | morph selectPrevious]. 
-		'<right>' -> [:anEvent :morph |  morph showChildList]. 
-		'<left>' -> [:anEvent :morph | morph showParentList]. 
+		'<tab>' -> [:anEvent :morph |  morph showChildList]. 
+		'<Shift-tab>' -> [:anEvent :morph | morph showParentList]. 
```

Comment:
Must Shift be capitalized?

Reply:
At least its capitalized in the original event.

Reply:
Okay, thanks.